js/src/dropdown.js

```diff
@@ -192,10 +187,10 @@ const Dropdown = (($) => {
           continue
         }
 
-        // remove backdrop only if the dropdown menu will be hidden
-        const backdrop = $(parent).find(Selector.BACKDROP)[0]
-        if (backdrop) {
-          backdrop.parentNode.removeChild(backdrop)
+        // if this is a touch-enabled device we remove the extra
+        // empty mouseover listeners we added for iOS support
+        if ('ontouchstart' in document.documentElement) {
+          $('body').children().off('mouseover', $.noop)
```

Comment:
IF we want to keep the explicit selector, and target the children (rather than going one level down), would it actually be better to do something like `$('body').children().on('mouseover', null, $.noop)` / `$('body').children().off('mouseover', null, $.noop)` ?